src/api-service/__app__/tasks/__init__.py

```diff
@@ -71,9 +72,8 @@ def get(req: func.HttpRequest) -> func.HttpResponse:
         task = Task.get_by_task_id(request.task_id)
         if isinstance(task, Error):
             return not_ok(task, context=request.task_id)
-        summary = Heartbeat.get_heartbeats(task.task_id)
-        task.heartbeats = summary
-
+        task.heartbeats = Heartbeat.get_heartbeats(task.task_id)
+        task.nodes = NodeTasks.get_node_assignments(request.task_id)
```

Comment:
![image](https://user-images.githubusercontent.com/244326/94619457-ce3b2d00-0261-11eb-9e7e-808c6369a4dd.png)

Reply:
The screenshot is super helpful here.